orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/util/PackageInfo.groovy

```diff
@@ -41,6 +44,8 @@ class PackageInfo {
     this.extractVersion = extractVersion
     this.mapper = mapper
     this.buildDetailExtractor = new BuildDetailExtractor()
+
+    filePattern = Pattern.compile("${stage?.context?.package}.*\\.${packageType}")
```

Comment:
I don't think you need to `stage?.context?.package` ... should be able to get away with `stage.context.package`.